src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports3.java

```diff
@@ -0,0 +1,44 @@
+/*
+UnusedImports
+processJavadoc = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
+
+import java.awt.AWTException;
+import java.awt.Rectangle; // violation 'Unused import - java.awt.Rectangle'
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent; // violation 'Unused import - java.awt.event.KeyEvent'
+
+public class InputUnusedImports3 extends java.awt.Robot {
+    private int delay;
+
+    protected InputUnusedImports3() throws AWTException {
+        super();
+    }
+
+    /**
+     * Press and release a key.
```

Comment:
`Press and release a key.` -> `Attention: usage of class by short name KeyEvent is not qualified as usage of type. Qualified usage is by javadoc tag 'link'`

Reply:
I have changed
done